src/WebJobs.Script.WebHost/Diagnostics/WebHostSystemLoggerProvider.cs

```diff
@@ -18,4 +18,4 @@ public WebHostSystemLoggerProvider(IEventGenerator eventGenerator, IEnvironment
         {
         }
     }
-}
+}
```

Comment:
nit: undo file changes that only touch the line ending

Reply:
Done.